packages/core/src/Stores/client-store.js

```diff
@@ -175,7 +175,7 @@ export default class ClientStore extends BaseStore {
      * 4. call statement, if the length is 0
      * 5. whenever has any transaction and start showing reality check
      *
-     * three:
+     * three: (we are not exaclt sure about this, so for now let ignore it, but we keep it here to investigate it later)
```

Comment:
Are we missing out on requirements here? are we going to investigate in the same PR?

nit: please fix the type `exaclt`

Reply:
@matin-binary fixed